utils/db/repos.ts

```diff
@@ -3,6 +3,36 @@ import conn from '.';
 import type { DbRepo, RepoIdentifier } from '../../types/repository';
 import { convert } from './converter';
 
+export const saveRepoIdentifierToDb = async (repos: RepoIdentifier[], topicId: string) => {
+	const insertRepoQuery = `
+		INSERT INTO repos (
+			repo_provider,
+			repo_owner,
+			repo_name,
+			install_id,
+			is_installed,
+			created_at
```

Comment:
You actually don't need to add this if it is just going to be "NOW()". This is already how it is set by default.

Reply:
done